herbert_core/admin/@opt_config_manager/private/find_comp_type_.m

```diff
@@ -56,17 +60,18 @@
     else
         n_profile = 4; % normal unix machine
     end
-
+    
     if hpc_computer
         n_profile=n_profile+1;
     end
     pc_type = types{n_profile};
-
+    %
+    if is_jenkins()
```

Comment:
the same short-circuit logic applies here; if it's a jenkins run all the other checks are unnecessary, so this should be the first clause.

Reply:
again -- not only checks but should deal with number of processors. It is not correct here, as was not nesessary, but Harry is doing something for this in other places, so this change for the future. I made appropriate comments to the function.